engine/src/test/java/org/hibernate/validator/test/internal/engine/methodvalidation/ClassLoaderMethodParameterConstraintsTest.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.test.internal.engine.methodvalidation;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import org.hibernate.validator.HibernateValidator;
+import org.hibernate.validator.HibernateValidatorConfiguration;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertNotNull;
+
+/**
+ * Integration test for {@link import org.hibernate.validator.internal.engine.ValidatorImpl} which tests that the ClassLoader of the
+ * base object and of classes in a parameter list do not affect validation.
+ * These tests attempt to simulate a web application that has its own class loader that
+ * does not preferentially delegate to a parent class loader.
+ *
+ * @author Chris Beckey &lt;cbeckey@paypal.com&gt;
+ */
+public class ClassLoaderMethodParameterConstraintsTest {
```

Comment:
I am not quite clear about the rationale behind this test; Did you run into any specific issue which you'd like to cover in the test suite? Is it really related to the rule relaxation?